### PR TITLE
Add the metrix to the ElixirTools

### PR DESCRIPTION
**Pull request by hylkealons**
#### :tophat: Problem
Metrix is to be shaed between multiple applications.

#### :pushpin: Solution
Add metrix to the Elixir Tools so it can be shared between multiple applications. 

What is missing is the tracer, I'd like to discuss with @Kitton how to implement this best.

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -23,6 +23,14 @@ use Mix.Config
 
 config :ex_aws, :sns, region: System.get_env("AWS_DEFAULT_REGION")
 
+config :pagantis_elixir_tools, ElixirTools.Metrix,
+  default_tags: %{},
+  adapter: ElixirTools.Metrix.Adapters.Ignore,
+  recurrent_metrics: [
+    {ElixirTools.Metrix.Recurring.ErlangStats, []}
+    # {PgPayments.Metrics.PendingOperations, []}
```
Comment (dabeat) [inline config/config.exs]: What's this comment about?

Comment (hylkealons) [reply config/config.exs]: copy pasta error!

**Comment by dabeat** [comment]
Looks good!

File: lib/json_api/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Prepare the router.ex, so all requests going to /api are JSON api and validated
 Use in all controllers the JsonApi.Controller:
 
 ```elixir
-  use PgPaymentsWeb, :controller
+  use MyAppWeb, :controller
```
Comment (dabeat) [inline lib/json_api/README.md]: :clap:

**Pull request merged**